packages/rocketchat-api/server/v1/users.js

```diff
@@ -273,7 +273,7 @@ RocketChat.API.v1.addRoute('users.createToken', { authRequired: true }, {
 
 RocketChat.API.v1.addRoute('users.getPreferences', { authRequired: true }, {
 	get() {
-		const user = this.userId;
+		const user = RocketChat.models.Users.findOneById(this.userId);
```

Comment:
@MarcosSpessatto we are going back to what we previously had here, which @rodrigok asked to change for `this.userId` Any particular reason?